### DISCUSSION TITLE
同步数据文件过大，占用C盘空间，该怎么办

### DISCUSSION THREAD

**Discussion by justinbread** [Chia-Network/chia-blockchain#10917] [Support]
如题，系统装在C盘，CHIA默认也是装在C盘，随着同步数据的增加，C盘空间马上要用完了，该怎么办呢？

**Comment by loppefaaret** [comment]
update to 1.3.1 and do a database upgrade to v2 - you might want to relocate the current database to a temporary drive, so you wont run out of space during the conversion - if you have a plotting SSD, i suggest using that while doing the upgrade - see https://github.com/Chia-Network/chia-blockchain/discussions/10845

**Comment by Jutean** [comment]
用修改iTunes备份的方法
先将.chia文件复制到其他盘
然后运行——CMD
mklink/j "C:\Users\用户名\.chia" "其他盘路径"
